templates/docs/examples/patterns/forms/checkbox-validation.html

```diff
@@ -0,0 +1,18 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Checkbox Validation{% endblock %}
+
+{% block standalone_css %}patterns_form-tick-elements{% endblock %}
+
+{% block content %}
+<form>
+    <div class=".p-form-validation is-error">
+            <label class="p-checkbox">
```

Comment:
You can add `is-required` to the checkbox element, and remove * from the text.

```suggestion
            <label class="p-checkbox is-required">
```